app/models/feedback/steps/purpose.rb

```diff
@@ -0,0 +1,19 @@
+module Feedback
+  module Steps
+    class Purpose < ::GITWizard::Step
+      OPTIONS = [
+        OpenStruct.new(id: "Give feedback about the website", value: "Give feedback about the website"),
+        OpenStruct.new(id: "Give feedback about signing up for an adviser", value: "Give feedback about signing up for an adviser"),
+        OpenStruct.new(id: "Give feedback about signing up for emails", value: "Give feedback about signing up for emails"),
+        OpenStruct.new(id: "Give feedback about signing up for an event", value: "Give feedback about signing up for an event"),
```

Comment:
@StackedPancakez Ideally we would add in another option at the end of the list

`Give feedback about booking a callback`